resources/plugins/test-client/webpack.config.js

```diff
@@ -17,19 +19,12 @@ module.exports = {
     path: path.resolve(__dirname, 'dist'),
     filename: 'client.js'
   },
-  module: {
-    rules: [
-      {
-        test: /\.js$/,
-        exclude: /node_modules/,
-        use: {
-          loader: 'babel-loader',
-          options: {
-            presets: [ '@babel/preset-react' ]
-          }
-        }
-      }
-    ]
-  },
-  devtool: 'cheap-module-source-map'
+  plugins: [
+    new CamundaModelerWebpackPlugin({
+      type: 'react'
+    }),
+    new CamundaModelerWebpackPlugin({
+      type: 'carbonReact'
+    })
```

Comment:
```suggestion
    new CamundaModelerWebpackPlugin()
```

Don't we just wanna roll with the default config? 🤔

Reply:
We could. But since this is just a test plugin, we're fine either way.